pagopa/pm/spec.json

```diff
@@ -1818,6 +1818,55 @@
           }
         ]
       }
+    },
+    "/v1/wallet/{idWallet}/payment-status": {
+      "put": {
+        "summary": "change the ability to pay",
+        "operationId": "changePayOption",
+        "parameters": [
+          {
+            "in": "path",
+            "name": "idWallet",
+            "type": "integer",
+            "required": true,
+            "description": "Numeric ID of the wallet"
+          },
+          {
+            "in": "body",
+            "name": "payRequest",
+            "description": "New Status required",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/WalletPaymentstatus"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "New wallet State",
+            "schema": {
+              "$ref": "#/definitions/WalletV2Response"
```

Comment:
Is it right that the return type is `WalletV2Response`?  
Atm we have `Wallet`, `WalletV2` and `WalletV3`. What will happen when we update to `WalletV3` and the answer to this call fails the check?